### DISCUSSION TITLE
batch slicing

### DISCUSSION THREAD

**Discussion by pca006132** [elalish/manifold#662] [General]
This looks interesting: https://lukeparry.uk/super-slicing-performance-in-3d-printing-with-pyslm/

I don't think there are users requiring batch slicing for now, but it might be something interesting that we can try when someone complains about slicing performance.

**Comment by elalish** [comment]
Yeah, that sounds like a lot of code for a little perf improvement, but maybe. I still want to get around to re-implementing my old straight skeleton algorithm - that gives a nice polygon to manifold "roof". One cool thing is then you can use `Slice` to get all the polygon offsets quickly instead of going through Clipper. Again, not sure who will be perf limited by 2D ops anyway.

**Reply by pca006132** [reply]
yeah, and it is a bit hard to see if the claimed performance improvement is actually that significant considering

> the implementation described in the post ignores sequential sorting of the generated line segments into correctly orientated connected paths that can construct polygons.

and IMO it is the part that cannot be parallelized easily.
